aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -167,113 +211,142 @@ def export(
     elif len(times) == 4:
         export_start, export_end, compress_start, compress_end = times
         EXPORT_LOGGER.debug(
-            'Exported in %6.2g s, compressed in %6.2g s, total: %6.2g s.', export_end - export_start,
-            compress_end - compress_start, compress_end - export_start
+            'Exported in %6.2g s, compressed in %6.2g s, total: %6.2g s.',
+            export_end - export_start,
+            compress_end - compress_start,
+            compress_end - export_start,
         )
     else:
         EXPORT_LOGGER.debug('No information about the timing of the export.')
 
     # Reset logging level
     if silent:
-        logging.disable(level=logging.NOTSET)
+        logging.disable(logging.NOTSET)
 
 
-def export_zip(entities=None, filename=None, use_compression=True, **kwargs):
+def export_zip(
+    entities: Optional[Iterable[Any]] = None,
+    filename: Optional[str] = None,
+    use_compression: bool = True,
+    **traversal_rules: bool,
+) -> Tuple[float, ...]:
```

Comment:
Couldn't that be 
```suggestion
) -> Tuple[float, float]:
```

Reply:
no because `export_tar` returns `Tuple[float, float, float, float]`, so you get an error later on